README.md

```diff
@@ -28,6 +28,7 @@ Project specific community blogs and presentations are at
 * [Argo Rollouts](https://github.com/argoproj/argo-rollouts/blob/master/README.md#community-blogs-and-presentations)
 
 ## Project Resources
+* [Argo Community Calendar](https://calendar.google.com/calendar/embed?src=argoproj@gmail.com)
```

Comment:
Can you also add a link to the public iCal format:

https://calendar.google.com/calendar/ical/argoproj%40gmail.com/public/basic.ics

This will allow users to add the iCal address to their calendars.